nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/server/SnifferManagerImpl.java

```diff
@@ -150,6 +152,7 @@ private <T extends Sniffer> List<T> getApplicableSniffers(DeploymentContext cont
                 continue;
             }
 
+            addSniffer:
```

Comment:
Couldn't it be refactored to avoid labels and "goto"?
https://www.baeldung.com/java-labeled-break

Reply:
Sure, but would require more time. I’ll see if I can find a quick way to refactor it.